packages/client/src/rtc/Publisher.ts

```diff
@@ -305,21 +305,26 @@ export class Publisher {
     const transceiver = this.pc
       .getTransceivers()
       .find((t) => t === this.transceiverRegistry[trackType] && t.sender.track);
+
     if (
       transceiver &&
       transceiver.sender.track &&
       (stopTrack
         ? transceiver.sender.track.readyState === 'live'
         : transceiver.sender.track.enabled)
     ) {
-      stopTrack
-        ? transceiver.sender.track.stop()
-        : (transceiver.sender.track.enabled = false);
+      // Since React Native follows the new Device API this is not needed
+      if (!isReactNative()) {
```

Comment:
Does that mean that RN doesn't stop publishing, just sends a mute state update to the SFU?

Reply:
Agh no, the purpose here is there was a repetition in the logic of muting here and at https://github.com/GetStream/stream-video-js/blob/main/packages/client/src/devices/InputMediaDeviceManager.ts#L163, hence the mute in the `muteTrack` function doesn't run at all. Ideally, it should run in only one place. Because of this, the enabled always remained false even after update.